clusterctl/testdata/delete-cluster-no-args-invalid-flag.golden

```diff
@@ -3,8 +3,12 @@ Usage:
   clusterctl delete cluster [flags]
 
 Flags:
+      --cluster string               The name of the kubeconfig cluster to use
```

Comment:
This help text isn't super clear without reading the code. Maybe "The name of the cluster to use in the kubeconfig file"?

Reply:
Your second option seems reasonable to me.

Reply:
Will follow up with another PR in kube core -- I have an open issue here:  https://github.com/kubernetes-sigs/cluster-api/issues/397